structlog_sentry/__init__.py

```diff
@@ -168,6 +168,26 @@ def _handle_breadcrumb(self, event_dict: EventDict) -> None:
             event, hint = self._get_breadcrumb_and_hint(event_dict)
             self._get_hub().add_breadcrumb(event, hint=hint)
 
+    def _get_level_value(
+        self,
+        level_name: str,
+        default_value: int = logging.NOTSET,
+    ) -> int:
+        """Get numeric value for the log level name given."""
+        level_name = level_name.upper()
+        try:
+            return getattr(logging, level_name)
+        except AttributeError:
+            pass
+
+        # Seems to be a custom log level
+        level = logging.getLevelName(level_name)
+        if isinstance(level, int):
+            return level
+
+        # Log level value not found
+        return default_value
```

Comment:
I think it would be better to fall back to the original attribute error if it is not a custom log level. Or what do you think?

Reply:
Changed to re-raise original exception